docs/fidesops/docs/guides/users.md

```diff
@@ -0,0 +1,52 @@
+# Admin UI
+
+We include some user-related endpoints for the Fidesops Admin UI. In this section, we'll cover:
+
+# I need permissions to be able to create a user. So do I create the first user?
```

Comment:
Looks like these items went out with `#` instead of a `-`.

Reply:
whoops!